vaas-app/src/vaas/vcl/tests/expected-vcl-4.0-with-mesh_service.vcl

```diff
@@ -11,13 +11,14 @@ backend default {
     .host = "127.0.0.1";
     .port = "30001";
 }
-sub use_director_director_with_envoy_support {
+sub use_director_director_with_mesh_service_support {
     set req.http.original-host = req.http.host;
-    set req.http.Host = "envoy_support";
+    set req.http.Host = "";
     unset req.http.X-Accept-Proto;
     set req.http.X-Accept-Proto = "https";
     unset req.http.X-VaaS-Prefix;
-    set req.http.X-VaaS-Prefix = "/envoy/support";
+    set req.http.X-VaaS-Prefix = "/mesh_service/support";
+    set req.backend_hint = default;
```

Comment:
I thought about putting that line in vcl_recv otherwise it will be duplicated in each use_director sub.